cfgov/unprocessed/js/modules/util/data-hook.js

```diff
@@ -60,8 +60,22 @@ function contains( element, value ) {
   return values.indexOf( value ) > -1 ? true : false;
 }
 
+/**
+ * @param {string} behavior
+ *  Behavior type used to find the element within the dom.
+ * @returns {HTMLNode} if it exists in the dom, null otherwise.
+ */
+function get( behavior ) {
+  var behaviorSelector =
+  standardType.JS_HOOK + '*=' + standardType.BEHAVIOR_PREFIX + behavior;
+  var behaviorElement = document.querySelector( '[' + behaviorSelector + ']' );
```

Comment:
It may help to pass in the scope, so instead of requiring querying on `document`, have that be the default that can be overridden by a passed in element. This would be helpful for behaviors that have several pieces and don't want to search the whole document again.